packages/library-legacy/test/connection.test.ts

```diff
@@ -4919,90 +4919,61 @@ describe('Connection', function () {
       const connection = new Connection(url, 'confirmed');
       const newAccount = PublicKey.unique();
 
-      let payerKeypair = new Keypair();
-      let testTokenMintPubkey: PublicKey;
-      let testOwnerKeypair: Keypair;
-      let testTokenAccountPubkey: PublicKey;
-      let testSignature: TransactionSignature;
+      const payerPublicKey = new PublicKey(
+        '6bffFjUMnnk3tXFJAS8pp9tz6m7oJAd27vm9rKqnLtVv',
+      );
+      const testTokenMintPubkey = new PublicKey(
+        '7MbpdfJa5xqwexkp6WUvkYHTPo4VgxYACDBNFWYLwCdo',
+      );
+      const testOwnerKeypair = Keypair.fromSecretKey(
+        new Uint8Array([
+          153, 120, 247, 45, 160, 119, 144, 219, 220, 209, 73, 91, 210, 102, 31,
+          136, 155, 12, 68, 27, 226, 215, 61, 214, 10, 245, 247, 180, 236, 63,
+          100, 202, 140, 247, 112, 54, 120, 32, 168, 118, 72, 115, 190, 34, 171,
+          126, 15, 119, 252, 173, 50, 173, 8, 10, 96, 239, 21, 32, 94, 67, 37,
+          43, 145, 249,
+        ]),
```

Comment:
I'm wondering if this should be a fixture somewhere, like `scripts/keypairs`.

Reply:
I at least wrote a comment to note what the pubkey is and where to find the fixture.